docs/README.md

```diff
@@ -105,11 +105,11 @@ Notes about Wine
 ----
 
 Wine is now partly supported. Wine integrated program all runs, and some windows programs and games now runs fine. Don't forget most Windows games use Direct3D, this may require a complete OpenGL driver and as high profile as possible (and gl4es with ES2 backend have issue with Wine for now). Also, vulkan is not wrapped on box86, so vk3d is not usable yet, even if supported by the hardware.
-Note: if you plan to use box86 with Wine on Raspberry Pi 3 or earlier, those models use a default OS that have a kernel with a 2/2 Split (meaning 2G of space for user program, and 2G of space for the Kernel). This is not compatible with Wine programs that needs to access memory > 2Gb address. So you'll need to reconfigure your kernel for a 3G/1G split. Use your favorite search engine for instructions on how to do that.
+Note: if you plan to use box86 with Wine on Raspberry Pi 3 or earlier, those models use a default OS that have a kernel with a 2/2 Split (meaning 2G of space for user program, and 2G of space for the Kernel). This is not compatible with Wine programs that needs to access memory > 2Gb address. So you'll need to reconfigure your kernel for a 3G/1G split. Wine on [pi-apps](https://github.com/Botspot/pi-apps/) lets you install a rebuilt kernel with memory split or switch to a experimental 64-bit kernel which also has a 3G/1G split.
 
```

Comment:
Same remark about PIapps limited to Pi4 for pi-apps advertisement here

Reply:
actually the pi-apps wine also works for pi3, it asks you if you want to change to 64bit kernel, compile 3g/1g kernel, or install precompiled 3g/1g kernel (but it may be outdated).